README.md

```diff
@@ -221,6 +221,13 @@ headers
 * `mix deps.get`
 * `mix test`
 
+## Publishing a new version
+
+To publish a new version:
+
+* Update the `version` string in `mix.exs` according to [SemVer](https://semver.org/spec/v2.0.0.html)
+* Run `mix hex.publish`
```

Comment:
Thoughts on moving this into a separate MD file, like a `RELEASE_PROCESS` file or something? That way most people won't encounter this when it's not relevant to their interests in reading the README.